config/src/test/scala/com/typesafe/config/impl/TokenizerTest.scala

```diff
@@ -239,15 +286,20 @@ class TokenizerTest extends TestUtils {
     def commentsHandledInVariousContexts() {
         tokenizerTest(List(tokenString("//bar")), "\"//bar\"")
         tokenizerTest(List(tokenString("#bar")), "\"#bar\"")
-        tokenizerTest(List(tokenUnquoted("bar"), tokenComment("comment")), "bar//comment")
-        tokenizerTest(List(tokenUnquoted("bar"), tokenComment("comment")), "bar#comment")
-        tokenizerTest(List(tokenInt(10), tokenComment("comment")), "10//comment")
-        tokenizerTest(List(tokenInt(10), tokenComment("comment")), "10#comment")
-        tokenizerTest(List(tokenDouble(3.14), tokenComment("comment")), "3.14//comment")
-        tokenizerTest(List(tokenDouble(3.14), tokenComment("comment")), "3.14#comment")
+        tokenizerTest(List(tokenUnquoted("bar"), tokenComment("comment", true)), "bar//comment")
+        tokenizerTest(List(tokenUnquoted("bar"), tokenComment("comment", false)), "bar#comment")
+        tokenizerTest(List(tokenInt(10), tokenComment("comment", true)), "10//comment")
+        tokenizerTest(List(tokenInt(10), tokenComment("comment", false)), "10#comment")
+        tokenizerTest(List(tokenDouble(3.14), tokenComment("comment", true)), "3.14//comment")
+        tokenizerTest(List(tokenDouble(3.14), tokenComment("comment", false)), "3.14#comment")
```

Comment:
here's where `tokenDoubleSlashComment("comment")` and `tokenHashComment("comment")` would be clearer than the boolean